lib/statusapi.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+
+	fcm "github.com/NaySoftware/go-fcm"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/params"
+)
+
+type StatusAPI interface {
```

Comment:
A few comments about this:
1. We don't need all methods here, just ones used in `library.go`,
2. It's possible to compose interfaces so we could have:
```go
type StatusAPIAccount interface {
  CreateAccount(password string) common.AccountInfo
  CreateChildAccount(parentAddress, password string) common.AccountInfo
  RecoverAccount(password, mnemonic string) common.AccountInfo
  VerifyAccountPassword(keyStoreDir, address, password string) (*keystore.Key, error)
  SelectAccount(address, password string) error
}

type StatusAPI interface {
  StatusAPIAccount
  // other interfaces...
}
```
it should make it easier to read and maintain.